packages/bpk-mixins/src/mixins/_typography.scss

```diff
@@ -591,6 +591,7 @@
 
 @mixin bpk-table__cell {
   padding: $bpk-spacing-sm;
+  overflow: hidden;
```

Comment:
obviously we would need to assess the blast radius of this change - as the mixin could be used in other components...